R/utils.R

```diff
@@ -5,4 +5,122 @@ get_file_type <- function(tracks){
   # character vector:
   #  - .bam$
   #  - .bw$
-}
\ No newline at end of file
+}
+
+
+#check that seqlevelsStyle of x and y match
+#' @noRd
+seqlevels_match <- function(x,y) {
+  
+	if(seqlevelsStyle(x) == seqlevelsStyle(y)){
```

Comment:
What if you change the function body to:

```
seqlevels_match <- function(x,y) {
   seqlevelsStyle(x) == seqlevelsStyle(y)
}
```

Since that expression evaluates to boolean already. Little cleaner.